wgpu-core/src/id.rs

```diff
@@ -128,7 +128,13 @@ impl<T> Clone for Id<T> {
 
 impl<T> fmt::Debug for Id<T> {
     fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
-        self.unzip().fmt(formatter)
+        let (index, epoch, backend) = self.unzip();
+        formatter
+            .debug_struct("Id")
+            .field("index", &index)
+            .field("epoch", &epoch)
+            .field("backend", &backend)
+            .finish()
```

Comment:
This is the only thing that might be an issue as this I believe, shows up when wgpu doesn't have a label for a buffer and if this ends up on multiple lines might have things harder to understand. Not a blocker though, going to merge anyway.